assets/src/edit-story/components/panels/animation/effectInput.js

```diff
@@ -75,6 +81,26 @@ function EffectInput({ effectProps, effectConfig, field, onChange }) {
           />
         </RangeContainer>
       );
+    case FIELD_TYPES.DIRECTION_PICKER:
+      return (
+        <DirectionRadioInput
+          directions={Object.values(DIRECTION)}
+          defaultChecked={
+            effectConfig[field] || effectProps[field].defaultValue
+          }
+          onChange={directionControlOnChange}
+        />
+      );
+    case FIELD_TYPES.ROTATION_PICKER:
+      return (
+        <DirectionRadioInput
+          directions={[DIRECTION.LEFT_TO_RIGHT, DIRECTION.RIGHT_TO_LEFT]}
+          defaultChecked={
+            effectConfig[field] || effectProps[field].defaultValue
+          }
+          onChange={directionControlOnChange}
+        />
```

Comment:
Only worry with this is eventually needing `CLOCKWISE` & `COUNTERCLOCKWISE` directions, but looking through this doesn't look like we need it at all rn and can address later when we do